Gordon360/Services/WellnessService.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using Gordon360.Models;
+using Gordon360.Models.ViewModels;
+using Gordon360.Repositories;
+using Gordon360.Exceptions.CustomExceptions;
+using System.Data.SqlClient;
+using Gordon360.Services.ComplexQueries;
+using System.Diagnostics;
+
+namespace Gordon360.Services
+{
+    public class WellnessService : IWellnessService
+    {
+       
+
+        public WellnessService()
+        {
+        }
+
+        /// <summary>
+        /// gets status of wellness check
+        /// </summary>
+        /// <param name="id">id</param>
+        /// <returns>boolean if found, null if not found</returns>
+
+        public IEnumerable<WellnessViewModel> GetStatus(string id)
+        {
+            var _unitOfWork = new UnitOfWork();
+            var query = _unitOfWork.AccountRepository.FirstOrDefault(x => x.gordon_id == id);
```

Comment:
All "var" in this new file should be "let".  We won't immediately change existing "var" to "let", but we shouldn't add new files that use "var".

Reply:
I'm not too sure that "let" is an actual thing in C#, I think your referencing not using var in React.